### PR TITLE
Update prefix-and-postfix-of-increment-operators.md

### PR DESCRIPTION
**Pull request by Stefan-Stojanovic**
Fix incorrect output in first code blocks comment

### CODE REVIEW

**Comment by enki-hq** [comment]
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

File: java/fundamentals/operators-ii/prefix-and-postfix-of-increment-operators.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Java supports 2 forms of shorter incrementation[1] of variables:
 ```java
 int i = 3;
 System.out.println(i++); 
-// 4
+// 3
```
Comment (kapnobatai137) [inline java/fundamentals/operators-ii/prefix-and-postfix-of-increment-operators.md:19]: `// 4` was the correct value

here is the code in a REPL

image

the value of `i` is incremented **after** it is printed when using postfix incrementation

**Review by kapnobatai137 — CHANGES_REQUESTED** [review]

File: java/fundamentals/operators-ii/prefix-and-postfix-of-increment-operators.md (hunk 1 above)
Comment (Stefan-Stojanovic) [reply java/fundamentals/operators-ii/prefix-and-postfix-of-increment-operators.md:19]: How is 4 correct?
when the outputs are 3 and 5?

Comment (Stefan-Stojanovic) [reply java/fundamentals/operators-ii/prefix-and-postfix-of-increment-operators.md:19]: You contradicted yourself in the last sentence.

> the value of i is incremented after it is printed when using postfix incrementation

The value of i = 3
the first print prints 3 and then it increments to 4
then the next print increments to 5 and then prints

Comment (kapnobatai137) [reply java/fundamentals/operators-ii/prefix-and-postfix-of-increment-operators.md:19]: oops, I'm blind. disregard my comment

**Review by kapnobatai137 — APPROVED** [review]

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |
### Changelog

| File Name | Error Message |
|:-:|:-:|
| Pull Request | Couldn't find a CHANGELOG entry for this pull request. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Comment by enki-hq** [comment]
### Topic

| Topic Name | Error Message |
|:-:|:-:|
| error-not-found | Topic should have at least one workout with the "workout" aspect. |

> If you notice anything wrong with these rules contact andrei@enki.com.

**Pull request merged**